rest/route/service.go

```diff
@@ -139,7 +139,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	// degraded_mode is used by Kanopy's alertmanager instance, which is only able to perform basic auth for REST, so it does not pass in user info
 	app.AddRoute("/degraded_mode").Version(2).Post().Wrap(requireAlertmanager).RouteHandler(makeSetDegradedMode())
 	app.AddRoute("/distros").Version(2).Get().Wrap(requireUser).RouteHandler(makeDistroRoute())
-	app.AddRoute("/distros/{distro_id}").Version(2).Get().Wrap(requireUser, editDistroSettings).RouteHandler(makeGetDistroByID())
+	app.AddRoute("/distros/{distro_id}").Version(2).Get().Wrap(requireUser).RouteHandler(makeGetDistroByID())
```

Comment:
there is a permission ViewDistroSettings we should probably use here instead (and for all distros) (should just be a line of code at the top)